R/tmb-methods.R

```diff
@@ -201,16 +198,38 @@ print.mmrm_tmb <- function(x,
 #' @param name of the component to be retrieved
 #' @param \dots ignored, for method compatibility
 #'
+#' @details Available `component()` names are as follows:
+#' * **call** The low-level function call which generated the model.
```

Comment:
maybe use 
```suggestion
#' - `call`: The low-level function call which generated the model.
```
way of writing this, just because other doc pages do like that